cl/lib/storage.py

```diff
@@ -50,39 +66,27 @@ class UUIDFileSystemStorage(FileSystemStorage):
     original file.
     """
 
-    def get_available_name(self, name, max_length=None):
+    def get_available_name(
+        self,
+        name: str,
+        max_length: Optional[int] = None,
+    ) -> str:
         dir_name, file_name = os.path.split(name)
         _, file_ext = os.path.splitext(file_name)
         return os.path.join(dir_name, uuid.uuid4().hex + file_ext)
 
 
 class AWSMediaStorage(S3Boto3Storage):
-    """Implements AWS file system storage.
-
-    We use AWSMediaStorage to upload LASC PDF files to AWS.
-    """
+    """Implements AWS file system storage with a few overrides"""
 
     location = ""
     AWS_DEFAULT_ACL = settings.AWS_DEFAULT_ACL
     file_overwrite = True
     custom_domain = False
 
-    def get_available_name(self, name, max_length=None):
-        """Generate usable file name for storage iterating if needed.
-
-        Returns a filename that is available in the storage mechanism,
-        possibly taking the provided filename into account
-
-        :param max_length: The name will not exceed max_length, if provided
-        :param name: File name of the object being saved.
-        :return: The filepath
-        """
-        dir_name, file_name = os.path.split(name)
-        file_root, file_ext = os.path.splitext(file_name)
-        count = itertools.count(1)
-        while self.exists(name):
-            # file_ext includes the dot.
-            name = os.path.join(
-                dir_name, "%s_%s%s" % (file_root, next(count), file_ext)
-            )
-        return name
+    def get_available_name(
+        self,
+        name: str,
+        max_length: Optional[int] = None,
+    ) -> str:
+        return get_name_by_incrementing(self, name, max_length)
```

Comment:
Why did you remove the docstrings?

Reply:
I think it'd be mostly redundant here, since it calls out to a single function that has a long docstring.